cfpb/cfgov-refresh
# Moving es templates into cfgov-refresh

Moving es templates into cfgov-refresh

## Additions

- Added template files for the main `/es/` navigation items.
- Added missing `/es` nemo assets..

## Removals

-

## Changes

- Modified `cfgov/cfgov/urls.py` to add routes for history file. 

## Testing
Enable the `ES_CONV_FLAG` flag.
1. Visit http://localhost:8000/es/ and verify the page renders. 
2. Visit http://localhost:8000/es/presentar-una-queja/ and verify that the page renders. 
3. Visit http://localhost:8000/es/quienes-somos/ and verify that the page renders.
4. Visit http://localhost:8000/es/hogar/` and verify that the page renders.
5. Visit http://localhost:8000/es/nuestra-historia/ and verify that the page renders.

## Screenshots


## Notes

-

## Todos

- Check the title and meta tags. 

## Checklist
